Sources/EdgeNetworkHandlers/NetworkResponseHandler.swift

```diff
@@ -152,7 +154,13 @@ class NetworkResponseHandler {
             if ignoreStorePayloads {
                 Log.debug(label: LOG_TAG, "Identities were reset recently, ignoring state:store payload for request with id: \(requestId)")
             } else {
-                handleStoreEventHandle(handle: eventHandle)
+                if let type = eventHandle.type {
+                    if EdgeConstants.JsonKeys.Response.EventHandleType.STORE == type.lowercased() {
+                        handleStoreEventHandle(handle: eventHandle)
+                    } else if EdgeConstants.JsonKeys.Response.EventHandleType.LOCATION_HINT == type {
+                        handleLocationHintHandle(handle: eventHandle)
```

Comment:
one type is lowercased and one is not, same in the handles below. is this expected?

Reply:
This is expected. The original code for handling the `state:store` responses lowercased the type before checking it. When updating the code I preserved the original logic, however I don't believe it is necessary to lower case the type.
Should we remove the lowercasing of the `state:store` type?

Reply:
The returned value is "state:store" all lowercase, and I think we should be consistent in all usages.